starlette_admin/templates/forms/ckeditor5.html

```diff
@@ -0,0 +1,10 @@
+<div class="{% if error %}{{ field.error_class }}{% endif %}">
+    <textarea name="{{ field.id }}" data-bs-toggle="autosize" class="{{ field.class_ }}
+            {% if error %}{{ field.error_class }}{% endif %}" {{ field.input_params() | safe }}
+            config="{{ field.config() }}"
```

Comment:
I think we can merge `config` with `field.input_params()` by overriding `input_params()` and adding the `config` attribute. A new form template is not neccessary